EBS_test: 0
acceleration: 0
autocross: 0
inspection: 0
manual_driving: 0
skidpad: 0
trackdrive: 0
